ui/webui/Makefile.am

```diff
@@ -0,0 +1,91 @@
+# Copyright (C) 2021  Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+SUBDIRS =
+
+# one example file in src/lib to check if it was already checked out
+LIB_TEST=src/lib/cockpit-po-plugin.js
+# stamp file to check if/when npm install ran
```

Comment:
I don't understand this comment.

Reply:
Line 21 I suppose?
This file, (package-lock.json) contains the detailed tree of the projects dependencies.
We use it as a file Makefile target, suggesting that if this file does not exist, then we need to run 'npm install' and re-generate the node_modules directory.